src/test/resources/alma-fix/99371123630706441.json

```diff
@@ -93,6 +93,10 @@
   } ],
   "extent" : "1 online resource (194 pages) : digital file(s).",
   "abstract" : [ "2012 wurde das Netzwerk-Projekt Landeskunde Nord ins Leben gerufen, dessen Ziel es ist, Forschung und Lehre zur Landeskunde des Deutschen als Fremdsprache, insbesondere in den nordischen Ländern, voranzutreiben. Das vorliegende Buch, Perspektive Nord, knüpft an den 2013 erschienenen Band Landeskunde Nord an und bietet Reflexionen wissenschaftstheoretischer, fachdidaktischer und inhaltlicher Aspekte, die für die Gestaltung der Landeskundelehre bzw. des Landeskundeunterrichts relevant sind. In ihrer Vielfalt sind die hier vorgelegten Konzepte und Ideen vor allem modernen kulturwissenschaftlichen Ansätzen verpflichtet, knüpfen an international geführte Diskussionen an und haben auch deshalb über die nordischen Länder hinaus Bedeutung. Dabei setzen die Autorinnen und Autoren auf autonomes und lebenslanges Lernen." ],
+  "license" : [ {
+    "id" : "https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode",
+    "label" : "creativecommons.org/licenses/by-nc-nd/4.0/legalcode"
```

Comment:
The label could be better. Apparently, we are currently not showing the field in the lobid UI, anyway. We will improve the labels when/if we add the field in the UI.